lib/api/liskApi.js

```diff
@@ -45,8 +45,8 @@ function LiskAPI (options) {
 
 	this.options = options;
 	this.ssl = options.ssl || false;
+	this.noRandomNode = options.noRandomNode || false;
 	this.testnet = options.testnet || false;
-	this.autoFindNode = options.autoFindNode || true;
```

Comment:
`noRandomNode` Not sure about negated logic.

`autoFindNode` was better in my opinion. Or: `randomPeer = true` for example.

```
if (this.randomPeer) {
```

Using `peer` instead of `node` is also more consistent with rest of code.